src/main/kotlin/com/cjcrafter/openai/AzureOpenAI.kt

```diff
@@ -26,13 +26,15 @@ class AzureOpenAI @JvmOverloads constructor(
 ) : OpenAI(apiKey, organization, client) {
 
     override fun buildRequest(request: Any, endpoint: String): Request {
+        val removedV1Endpoint = endpoint.removePrefix("v1/") // temporary fix for Azure, as it doesn't support v1/ in the url
```

Comment:
Instead of removing the prefix from the endpoint, a slightly more version stable method would be to have overridable functions for endpoints in the `OpenAI` class. This way, if OpenAI moves something to V2 we can easily change that in code with creating clutter anywhere else. 

`AzureOpenAI` can override these methods with their own endpoints, and can even throw an exception for unsupported endpoints.